docs/02-deployment-guide.md

```diff
@@ -176,20 +192,23 @@ if you run the Docker container multiple times you will still have details of th
 If you add a command on the end, you can run a specific script like this:
 
 ```shell
-sleeper deployment test/deployAll/deployTest.sh myinstanceid myvpc mysubnet
+sleeper builder sleeper/scripts/test/deployAll/deployTest.sh myinstanceid myvpc mysubnet
 ```
 
+There is also a [Nix shell](https://nixos.org/) file at the root of the repository, which can give you a shell with all dependencies required
+to build, deploy, and interact with instances.
+To enter the Nix shell you can run the following command
+
 ### Manual Deployment
 
 For Sleeper to be deployed manually, some resources have to be uploaded to AWS first:
 the jar files need to be uploaded to an S3 bucket, and some Docker images
 need to be uploaded to an ECR repository.
 
-These instructions will assume you're using a development environment, so see [the dev guide](11-dev-guide.md) for how
-to set that up. You can also use the `sleeper builder` CLI command to get a shell in a suitable environment, if you have
-the CLI configured and authenticated with AWS.
+These instructions will assume you start in the project root directory and you're using a development environment
+(see [Installing prerequesite software](02-deployment-guide.md#install-prerequisite-software) for how to set that up).
```

Comment:
It looks like some information has been removed here. The link is incorrect for how to set up a development environment, and there's a typo at "prerequesite". This paragraph and the one after could be reverted?